<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstrumentController</class>
 <widget class="QWidget" name="InstrumentController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="freq_horizontal_slider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="freq_min_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="freq_max_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="freq_line_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="volume_dial">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="volume_label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Volume</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>77</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PLAY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bpm_line_edit">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>78</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="bpm_label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>81</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BPM:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="instrument_combo_box">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>78</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="instrument_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Instrument:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
